features/call/src/main/kotlin/io/element/android/features/call/ui/ElementCallActivity.kt

```diff
@@ -63,6 +66,26 @@ class ElementCallActivity : NodeComponentActivity(), CallScreenNavigator {
             }
             context.startActivity(intent)
         }
+
+        /**
+         * Eventually start the ElementCallActivity, and return true if it's the case.
```

Comment:
This is a bit vague. Maybe 'Start the `ElementCallActivity' if the intent contains a valid URL, return `true` if it's the case'?

Reply:
Yes, thanks fa4ad4a9342b12a57424599736ec7026088a3442